Sync `MON_DATA_x` name and struct names with pokeheartgold

AsparagusEduardo opened the pull request:
Matching PR: ___

- Fixes https://github.com/pret/pokeplatinum/issues/640
- Changed enums/constant names:
  ```
  pokeplatinum                                | PR                                  | pokeheartgold
                                              |                                     |
  MON_DATA_IS_PARTY_DECRYPTED                 | MON_DATA_IS_PARTY_DECRYPTED         | MON_DATA_PARTY_LOCK
  MON_DATA_IS_BOX_DECRYPTED                   | MON_DATA_IS_BOX_DECRYPTED           | MON_DATA_BOX_LOCK
  MON_DATA_IS_DATA_INVALID                    | MON_DATA_CHECKSUM_FAILED            | MON_DATA_CHECKSUM_FAILED
  MON_DATA_OT_ID                              | MON_DATA_OT_ID                      | MON_DATA_OTID
  MON_DATA_EXP                                | MON_DATA_EXPERIENCE                 | MON_DATA_EXPERIENCE
  MON_DATA_MARKS                              | MON_DATA_MARKINGS                   | MON_DATA_MARKINGS
  MON_DATA_LANGUAGE                           | MON_DATA_LANGUAGE                   | MON_DATA_GAME_LANGUAGE
  MON_DATA_HOENN_COOL_RIBBON                  | MON_DATA_COOL_RIBBON                | MON_DATA_HOENN_COOL_RIBBON
  MON_DATA_HOENN_COOL_RIBBON_SUPER            | MON_DATA_COOL_RIBBON_SUPER          | MON_DATA_HOENN_COOL_RIBBON_SUPER
  MON_DATA_HOENN_COOL_RIBBON_HYPER            | MON_DATA_COOL_RIBBON_HYPER          | MON_DATA_HOENN_COOL_RIBBON_HYPER
  MON_DATA_HOENN_COOL_RIBBON_MASTER           | MON_DATA_COOL_RIBBON_MASTER         | MON_DATA_HOENN_COOL_RIBBON_MASTER
  MON_DATA_HOENN_BEAUTY_RIBBON                | MON_DATA_BEAUTY_RIBBON              | MON_DATA_HOENN_BEAUTY_RIBBON
  MON_DATA_HOENN_BEAUTY_RIBBON_SUPER          | MON_DATA_BEAUTY_RIBBON_SUPER        | MON_DATA_HOENN_BEAUTY_RIBBON_SUPER
  MON_DATA_HOENN_BEAUTY_RIBBON_HYPER          | MON_DATA_BEAUTY_RIBBON_HYPER        | MON_DATA_HOENN_BEAUTY_RIBBON_HYPER
  MON_DATA_HOENN_BEAUTY_RIBBON_MASTER         | MON_DATA_BEAUTY_RIBBON_MASTER       | MON_DATA_HOENN_BEAUTY_RIBBON_MASTER
  MON_DATA_HOENN_CUTE_RIBBON                  | MON_DATA_CUTE_RIBBON                | MON_DATA_HOENN_CUTE_RIBBON
  MON_DATA_HOENN_CUTE_RIBBON_SUPER            | MON_DATA_CUTE_RIBBON_SUPER          | MON_DATA_HOENN_CUTE_RIBBON_SUPER
  MON_DATA_HOENN_CUTE_RIBBON_HYPER            | MON_DATA_CUTE_RIBBON_HYPER          | MON_DATA_HOENN_CUTE_RIBBON_HYPER
  MON_DATA_HOENN_CUTE_RIBBON_MASTER           | MON_DATA_CUTE_RIBBON_MASTER         | MON_DATA_HOENN_CUTE_RIBBON_MASTER
  MON_DATA_HOENN_SMART_RIBBON                 | MON_DATA_SMART_RIBBON               | MON_DATA_HOENN_SMART_RIBBON
  MON_DATA_HOENN_SMART_RIBBON_SUPER           | MON_DATA_SMART_RIBBON_SUPER         | MON_DATA_HOENN_SMART_RIBBON_SUPER
  MON_DATA_HOENN_SMART_RIBBON_HYPER           | MON_DATA_SMART_RIBBON_HYPER         | MON_DATA_HOENN_SMART_RIBBON_HYPER
  MON_DATA_HOENN_SMART_RIBBON_MASTER          | MON_DATA_SMART_RIBBON_MASTER        | MON_DATA_HOENN_SMART_RIBBON_MASTER
  MON_DATA_HOENN_TOUGH_RIBBON                 | MON_DATA_TOUGH_RIBBON               | MON_DATA_HOENN_TOUGH_RIBBON
  MON_DATA_HOENN_TOUGH_RIBBON_SUPER           | MON_DATA_TOUGH_RIBBON_SUPER         | MON_DATA_HOENN_TOUGH_RIBBON_SUPER
  MON_DATA_HOENN_TOUGH_RIBBON_HYPER           | MON_DATA_TOUGH_RIBBON_HYPER         | MON_DATA_HOENN_TOUGH_RIBBON_HYPER
  MON_DATA_HOENN_TOUGH_RIBBON_MASTER          | MON_DATA_TOUGH_RIBBON_MASTER        | MON_DATA_HOENN_TOUGH_RIBBON_MASTER
  MON_DATA_SINNOH_SUPER_COOL_RIBBON           | MON_DATA_SUPER_COOL_RIBBON          | MON_DATA_COOL_RIBBON
  MON_DATA_SINNOH_SUPER_COOL_RIBBON_SUPER     | MON_DATA_SUPER_COOL_RIBBON_SUPER    | MON_DATA_COOL_RIBBON_SUPER
  MON_DATA_SINNOH_SUPER_COOL_RIBBON_HYPER     | MON_DATA_SUPER_COOL_RIBBON_HYPER    | MON_DATA_COOL_RIBBON_HYPER
  MON_DATA_SINNOH_SUPER_COOL_RIBBON_MASTER    | MON_DATA_SUPER_COOL_RIBBON_MASTER   | MON_DATA_COOL_RIBBON_MASTER
  MON_DATA_SINNOH_SUPER_BEAUTY_RIBBON         | MON_DATA_SUPER_BEAUTY_RIBBON        | MON_DATA_BEAUTY_RIBBON
  MON_DATA_SINNOH_SUPER_BEAUTY_RIBBON_SUPER   | MON_DATA_SUPER_BEAUTY_RIBBON_SUPER  | MON_DATA_BEAUTY_RIBBON_SUPER
  MON_DATA_SINNOH_SUPER_BEAUTY_RIBBON_HYPER   | MON_DATA_SUPER_BEAUTY_RIBBON_HYPER  | MON_DATA_BEAUTY_RIBBON_HYPER
  MON_DATA_SINNOH_SUPER_BEAUTY_RIBBON_MASTER  | MON_DATA_SUPER_BEAUTY_RIBBON_MASTER | MON_DATA_BEAUTY_RIBBON_MASTER
  MON_DATA_SINNOH_SUPER_CUTE_RIBBON           | MON_DATA_SUPER_CUTE_RIBBON          | MON_DATA_CUTE_RIBBON
  MON_DATA_SINNOH_SUPER_CUTE_RIBBON_SUPER     | MON_DATA_SUPER_CUTE_RIBBON_SUPER    | MON_DATA_CUTE_RIBBON_SUPER
  MON_DATA_SINNOH_SUPER_CUTE_RIBBON_HYPER     | MON_DATA_SUPER_CUTE_RIBBON_HYPER    | MON_DATA_CUTE_RIBBON_HYPER
  MON_DATA_SINNOH_SUPER_CUTE_RIBBON_MASTER    | MON_DATA_SUPER_CUTE_RIBBON_MASTER   | MON_DATA_CUTE_RIBBON_MASTER
  MON_DATA_SINNOH_SUPER_SMART_RIBBON          | MON_DATA_SUPER_SMART_RIBBON         | MON_DATA_SMART_RIBBON
  MON_DATA_SINNOH_SUPER_SMART_RIBBON_SUPER    | MON_DATA_SUPER_SMART_RIBBON_SUPER   | MON_DATA_SMART_RIBBON_SUPER
  MON_DATA_SINNOH_SUPER_SMART_RIBBON_HYPER    | MON_DATA_SUPER_SMART_RIBBON_HYPER   | MON_DATA_SMART_RIBBON_HYPER
  MON_DATA_SINNOH_SUPER_SMART_RIBBON_MASTER   | MON_DATA_SUPER_SMART_RIBBON_MASTER  | MON_DATA_SMART_RIBBON_MASTER
  MON_DATA_SINNOH_SUPER_TOUGH_RIBBON          | MON_DATA_SUPER_TOUGH_RIBBON         | MON_DATA_TOUGH_RIBBON
  MON_DATA_SINNOH_SUPER_TOUGH_RIBBON_SUPER    | MON_DATA_SUPER_TOUGH_RIBBON_SUPER   | MON_DATA_TOUGH_RIBBON_SUPER
  MON_DATA_SINNOH_SUPER_TOUGH_RIBBON_HYPER    | MON_DATA_SUPER_TOUGH_RIBBON_HYPER   | MON_DATA_TOUGH_RIBBON_HYPER
  MON_DATA_SINNOH_SUPER_TOUGH_RIBBON_MASTER   | MON_DATA_SUPER_TOUGH_RIBBON_MASTER  | MON_DATA_TOUGH_RIBBON_MASTER
  MON_DATA_HOENN_CHAMPION_RIBBON              | MON_DATA_CHAMPION_RIBBON            | MON_DATA_HOENN_CHAMPION_RIBBON
  MON_DATA_HOENN_WINNING_RIBBON               | MON_DATA_WINNING_RIBBON             | MON_DATA_HOENN_WINNING_RIBBON
  MON_DATA_HOENN_VICTORY_RIBBON               | MON_DATA_VICTORY_RIBBON             | MON_DATA_HOENN_VICTORY_RIBBON
  MON_DATA_HOENN_ARTIST_RIBBON                | MON_DATA_ARTIST_RIBBON              | MON_DATA_HOENN_ARTIST_RIBBON
  MON_DATA_HOENN_EFFORT_RIBBON                | MON_DATA_EFFORT_RIBBON              | MON_DATA_HOENN_EFFORT_RIBBON
  MON_DATA_HOENN_MARINE_RIBBON                | MON_DATA_MARINE_RIBBON              | MON_DATA_HOENN_MARINE_RIBBON
  MON_DATA_HOENN_LAND_RIBBON                  | MON_DATA_LAND_RIBBON                | MON_DATA_HOENN_LAND_RIBBON
  MON_DATA_HOENN_SKY_RIBBON                   | MON_DATA_SKY_RIBBON                 | MON_DATA_HOENN_SKY_RIBBON
  MON_DATA_HOENN_COUNTRY_RIBBON               | MON_DATA_COUNTRY_RIBBON             | MON_DATA_HOENN_COUNTRY_RIBBON
  MON_DATA_HOENN_NATIONAL_RIBBON              | MON_DATA_NATIONAL_RIBBON            | MON_DATA_HOENN_NATIONAL_RIBBON
  MON_DATA_HOENN_EARTH_RIBBON                 | MON_DATA_EARTH_RIBBON               | MON_DATA_HOENN_EARTH_RIBBON
  MON_DATA_HOENN_WORLD_RIBBON                 | MON_DATA_WORLD_RIBBON               | MON_DATA_HOENN_WORLD_RIBBON
  MON_DATA_MET_GAME                           | MON_DATA_MET_GAME                   | MON_DATA_GAME_VERSION
  MON_DATA_DUMMY_B_1                          | MON_DATA_UNUSED_113                 | MON_DATA_RESERVED_113
  MON_DATA_DUMMY_B_2                          | MON_DATA_UNUSED_114                 | MON_DATA_RESERVED_114
  MON_DATA_MET_GAME_REGION_CODE               | MON_DATA_UNUSED_121                 | MON_DATA_UNK_121
  MON_DATA_CONTEST_RIBBON_DUMMY               | MON_DATA_UNUSED_RIBBON_143          | MON_DATA_SINNOH_RIBBON_143
  MON_DATA_OTNAME                             | MON_DATA_OT_NAME                    | MON_DATA_OT_NAME
  MON_DATA_OTNAME_STRBUF                      | MON_DATA_OT_NAME_STRING             | MON_DATA_OT_NAME_2
  MON_DATA_SINNOH_ABILITY_RIBBON              | MON_DATA_ABILITY_RIBBON             | MON_DATA_ABILITY_RIBBON
  MON_DATA_SINNOH_GREAT_ABILITY_RIBBON        | MON_DATA_GREAT_ABILITY_RIBBON       | MON_DATA_GREAT_ABILITY_RIBBON
  MON_DATA_SINNOH_DOUBLE_ABILITY_RIBBON       | MON_DATA_DOUBLE_ABILITY_RIBBON      | MON_DATA_DOUBLE_ABILITY_RIBBON
  MON_DATA_SINNOH_MULTI_ABILITY_RIBBON        | MON_DATA_MULTI_ABILITY_RIBBON       | MON_DATA_MULTI_ABILITY_RIBBON
  MON_DATA_SINNOH_PAIR_ABILITY_RIBBON         | MON_DATA_PAIR_ABILITY_RIBBON        | MON_DATA_PAIR_ABILITY_RIBBON
  MON_DATA_SINNOH_WORLD_ABILITY_RIBBON        | MON_DATA_WORLD_ABILITY_RIBBON       | MON_DATA_WORLD_ABILITY_RIBBON
  MON_DATA_SINNOH_ALERT_RIBBON                | MON_DATA_ALERT_RIBBON               | MON_DATA_ALERT_RIBBON
  MON_DATA_SINNOH_SHOCK_RIBBON                | MON_DATA_SHOCK_RIBBON               | MON_DATA_SHOCK_RIBBON
  MON_DATA_SINNOH_DOWNCAST_RIBBON             | MON_DATA_DOWNCAST_RIBBON            | MON_DATA_DOWNCAST_RIBBON
  MON_DATA_SINNOH_CARELESS_RIBBON             | MON_DATA_CARELESS_RIBBON            | MON_DATA_CARELESS_RIBBON
  MON_DATA_SINNOH_RELAX_RIBBON                | MON_DATA_RELAX_RIBBON               | MON_DATA_RELAX_RIBBON
  MON_DATA_SINNOH_SNOOZE_RIBBON               | MON_DATA_SNOOZE_RIBBON              | MON_DATA_SNOOZE_RIBBON
  MON_DATA_SINNOH_SMILE_RIBBON                | MON_DATA_SMILE_RIBBON               | MON_DATA_SMILE_RIBBON
  MON_DATA_SINNOH_GORGEOUS_RIBBON             | MON_DATA_GORGEOUS_RIBBON            | MON_DATA_GORGEOUS_RIBBON
  MON_DATA_SINNOH_ROYAL_RIBBON                | MON_DATA_ROYAL_RIBBON               | MON_DATA_ROYAL_RIBBON
  MON_DATA_SINNOH_GORGEOUS_ROYAL_RIBBON       | MON_DATA_GORGEOUS_ROYAL_RIBBON      | MON_DATA_GORGEOUS_ROYAL_RIBBON
  MON_DATA_SINNOH_FOOTPRINT_RIBBON            | MON_DATA_FOOTPRINT_RIBBON           | MON_DATA_FOOTPRINT_RIBBON
  MON_DATA_SINNOH_RECORD_RIBBON               | MON_DATA_RECORD_RIBBON              | MON_DATA_RECORD_RIBBON
  MON_DATA_SINNOH_HISTORY_RIBBON              | MON_DATA_HISTORY_RIBBON             | MON_DATA_HISTORY_RIBBON
  MON_DATA_SINNOH_LEGEND_RIBBON               | MON_DATA_LEGEND_RIBBON              | MON_DATA_LEGEND_RIBBON
  MON_DATA_SINNOH_RED_RIBBON                  | MON_DATA_RED_RIBBON                 | MON_DATA_RED_RIBBON
  MON_DATA_SINNOH_GREEN_RIBBON                | MON_DATA_GREEN_RIBBON               | MON_DATA_GREEN_RIBBON
  MON_DATA_SINNOH_BLUE_RIBBON                 | MON_DATA_BLUE_RIBBON                | MON_DATA_BLUE_RIBBON
  MON_DATA_SINNOH_FESTIVAL_RIBBON             | MON_DATA_FESTIVAL_RIBBON            | MON_DATA_FESTIVAL_RIBBON
  MON_DATA_SINNOH_CARNIVAL_RIBBON             | MON_DATA_CARNIVAL_RIBBON            | MON_DATA_CARNIVAL_RIBBON
  MON_DATA_SINNOH_CLASSIC_RIBBON              | MON_DATA_CLASSIC_RIBBON             | MON_DATA_CLASSIC_RIBBON
  MON_DATA_SINNOH_PREMIER_RIBBON              | MON_DATA_PREMIER_RIBBON             | MON_DATA_PREMIER_RIBBON
  MON_DATA_SINNOH_RIBBON_DUMMY                | MON_DATA_UNUSED_RIBBON_53           | MON_DATA_SINNOH_RIBBON_53
  MON_DATA_SPECIES_EGG                        | MON_DATA_SPECIES_OR_EGG             | MON_DATA_SPECIES_OR_EGG
  MON_DATA_EGG_EXISTS                         | MON_DATA_SANITY_IS_EGG              | MON_DATA_SANITY_IS_EGG
  MON_DATA_MOVE1_MAX_PP                       | MON_DATA_MOVE1_MAX_PP               | MON_DATA_MOVE1MAXPP
  MON_DATA_MOVE2_MAX_PP                       | MON_DATA_MOVE2_MAX_PP               | MON_DATA_MOVE2MAXPP
  MON_DATA_MOVE3_MAX_PP                       | MON_DATA_MOVE3_MAX_PP               | MON_DATA_MOVE3MAXPP
  MON_DATA_MOVE4_MAX_PP                       | MON_DATA_MOVE4_MAX_PP               | MON_DATA_MOVE4MAXPP
  MON_DATA_MOVE1_CUR_PP                       | MON_DATA_MOVE1_CUR_PP               | MON_DATA_MOVE1PP
  MON_DATA_MOVE2_CUR_PP                       | MON_DATA_MOVE2_CUR_PP               | MON_DATA_MOVE2PP
  MON_DATA_MOVE3_CUR_PP                       | MON_DATA_MOVE3_CUR_PP               | MON_DATA_MOVE3PP
  MON_DATA_MOVE4_CUR_PP                       | MON_DATA_MOVE4_CUR_PP               | MON_DATA_MOVE4PP
  MON_DATA_MOVE1_PP_UPS                       | MON_DATA_MOVE1_PP_UPS               | MON_DATA_MOVE1PPUP
  MON_DATA_MOVE2_PP_UPS                       | MON_DATA_MOVE2_PP_UPS               | MON_DATA_MOVE2PPUP
  MON_DATA_MOVE3_PP_UPS                       | MON_DATA_MOVE3_PP_UPS               | MON_DATA_MOVE3PPUP
  MON_DATA_MOVE4_PP_UPS                       | MON_DATA_MOVE4_PP_UPS               | MON_DATA_MOVE4PPUP
  MON_DATA_NICKNAME_STRBUF_AND_FLAG           | MON_DATA_NICKNAME_STRING_AND_FLAG   | MON_DATA_NICKNAME_STRING_COMPARE
  MON_DATA_NICKNAME_STRBUF                    | MON_DATA_NICKNAME_STRING            | MON_DATA_NICKNAME_STRING
  MON_DATA_NIDORAN_HAS_NICKNAME               | MON_DATA_NO_PRINT_GENDER            | MON_DATA_NO_PRINT_GENDER
  MON_DATA_COMBINED_IVS                       | MON_DATA_COMBINED_IVS               | MON_DATA_IVS_WORD
  MON_DATA_MET_YEAR                           | MON_DATA_EGG_YEAR                   | MON_DATA_EGG_MET_YEAR
  MON_DATA_MET_MONTH                          | MON_DATA_EGG_MONTH                  | MON_DATA_EGG_MET_MONTH
  MON_DATA_MET_DAY                            | MON_DATA_EGG_DAY                    | MON_DATA_EGG_MET_DAY
  MON_DATA_HATCH_YEAR                         | MON_DATA_MET_YEAR                   | MON_DATA_MET_YEAR
  MON_DATA_HATCH_MONTH                        | MON_DATA_MET_MONTH                  | MON_DATA_MET_MONTH
  MON_DATA_HATCH_DAY                          | MON_DATA_MET_DAY                    | MON_DATA_MET_DAY
  MON_DATA_MET_LOCATION                       | MON_DATA_EGG_LOCATION               | MON_DATA_EGG_MET_LOCATION
  MON_DATA_HATCH_LOCATION                     | MON_DATA_MET_LOCATION               | MON_DATA_MET_LOCATION
  MON_DATA_FATEFUL_MET_LOCATION               | MON_DATA_HGSS_EGG_LOCATION          | MON_DATA_HGSS_EGG_MET_LOCATION
  MON_DATA_FATEFUL_HATCH_LOCATION             | MON_DATA_HGSS_NET_LOCATION          | MON_DATA_HGSS_MET_LOCATION
  MON_DATA_STATUS_CONDITION                   | MON_DATA_STATUS                     | MON_DATA_STATUS
  MON_DATA_BALL_CAPSULE_ID                    | MON_DATA_BALL_CAPSULE_ID            | MON_DATA_CAPSULE
  MON_DATA_CURRENT_HP                         | MON_DATA_CUR_HP                     | MON_DATA_HP
  MON_DATA_MAX_HP                             | MON_DATA_MAX_HP                     | MON_DATA_MAXHP
  MON_DATA_SP_ATK                             | MON_DATA_SP_ATK                     | MON_DATA_SPATK
  MON_DATA_SP_DEF                             | MON_DATA_SP_DEF                     | MON_DATA_SPDEF
  MON_DATA_MAIL                               | MON_DATA_MAIL                       | MON_DATA_MAIL_STRUCT
  MON_DATA_BALL_CAPSULE                       | MON_DATA_BALL_CAPSULE               | MON_DATA_SEAL_COORDS
  ```
  Every other one was already synced.
- 
- Changed structs:
  - `BoxPokemon`
    ```
    pokeplatinum    | PR              | pokeheartgold
                    |                 |
    personality     | personality     | pid
    partyDecrypted  | partyDecrypted  | party_lock
    boxDecrypted    | boxDecrypted    | box_lock
    invalidData     | checksumFailed  | checksum_fail
    padding_04_3    | unused          | Unused
    dataBlocks      | dataBlocks      | substructs
    ```
    - `PokemonDataBlockA`
      ```
      pokeplatinum  | PR        | pokeheartgold
                    |           |
      marks         | markings  | markings
      speedEV       | speedEV   | spdEV
      spAtkEV       | spAtkEV   | spatkEV
      spDefEV       | spDefEV   | spdefEV
      cool          | cool      | coolStat
      beauty        | beauty    | beautyStat
      cute          | cute      | cuteStat
      smart         | smart     | smartStat
      tough         | tough     | toughStat
      sinnohRibbons | ribbonsA  | sinnohRibbons
      ```
    - `PokemonDataBlockB`
      ```
      pokeplatinum          | PR                      | pokeheartgold
                            |                         |
      moveCurrentPPs        | moveCurrentPP           | movePP
      movePPUps             | movePPUps               | movePpUps
      speedIV               | speedIV                 | spdIV
      spAtkIV               | spAtkIV                 | spatkIV
      spDefIV               | spDefIV                 | spdefIV
      form                  | form                    | alternateForm
      fatefulMetLocation    | PlatHGSS_EggLocation    | Platinum_EggLocation
      fatefulHatchLocation  | PlatHGSS_MetLocation    | Platinum_MetLocation
      dummy_19              | unused1                 | Unused
      dummy_1A              | unused2                 | Unused
      hoennRibbons          | ribbonsB                | ribbonFlags
      hasNickname           | hasNickname             | isNicknamed
      ```
    - `PokemonDataBlockC`
      ```
      contestRibbons | ribbonsC | sinnohRibbons2
      ```
    - `PokemonDataBlockD`
      ```
      metLocation   | DP_EggLocation  | DP_EggLocation
      hatchLocation | DP_MetLocation  | DP_MetLocation
      otName        | otName          | otTrainerName
      metYear       | eggYear         | dateEggReceived[0]
      metMonth      | eggMonth        | dateEggReceived[1]
      metDay        | eggDay          | dateEggReceived[2]
      hatchYear     | metYear         | dateMet[0]
      hatchMonth    | metMonth        | dateMet[1]
      hatchDay      | metDay          | dateMet[2]
      ```
  - `PartyPokemon`/`PARTYMON`
    ```
    pokeplatinum  | PR            | pokeheartgold
                  |               |
    PartyPokemon  | PartyPokemon  | PARTYMON
    ballCapsuleID | ballCapsuleID | capsule
    maxHP         | maxHP         | maxHp
    ```

AsparagusEduardo commented:
> The names here are incorrect in all repos and the PR; Super Contests classify their rank with `Great`, `Ultra`, and `Master`: 

Fixed the description as well as adding * markers to indicate what names were changed for each repo

lhearachel commented:
```
*fatefulMetLocation   | PlatHGSS_EggLocation    | *Platinum_EggLocation
*fatefulHatchLocation | PlatHGSS_MetLocation    | *Platinum_MetLocation

*metLocation    | DP_EggLocation  |  DP_EggLocation
*hatchLocation  | DP_MetLocation  |  DP_MetLocation
```

For these, I feel that a format similar to `eggLocation_DP` would be ideal. I would also suggest `Pt` instead of `Plat`, because the format is an accepted acronym for the generation's games.

```
*hoennRibbons         | ribbonsB                | *ribbonFlags
```

Would `ribbonsGBA` and `ribbonsDS` work, denoting their origin generation? Or is that too confusing?

AsparagusEduardo commented:
As requested by @red031000 on pokeheart's side, the following changes have been made:
- Changed enums/constant names:
  ```
  pokeplatinum                                | PR                                  | pokeheartgold
                                              |                                     |
   *MON_DATA_CURRENT_HP                       | MON_DATA_HP                         | MON_DATA_HP
   *MON_DATA_MOVE1_CUR_PP                     | MON_DATA_MOVE1_PP                   | *MON_DATA_MOVE1PP
   *MON_DATA_MOVE2_CUR_PP                     | MON_DATA_MOVE2_PP                   | *MON_DATA_MOVE2PP
   *MON_DATA_MOVE3_CUR_PP                     | MON_DATA_MOVE3_PP                   | *MON_DATA_MOVE3PP
   *MON_DATA_MOVE4_CUR_PP                     | MON_DATA_MOVE4_PP                   | *MON_DATA_MOVE4PP
  ```
  - Removed redundant comments in PokemonData struct defs

lhearachel commented:
The proposed changes are fine by me.

AsparagusEduardo commented:
Merged on the HG side :D
